src/main/java/Katheryne.java

```diff
@@ -0,0 +1,91 @@
+import java.util.*;
+
+public class Katheryne {
+    public static void main(String[] args) throws KatheryneExceptions {
```

Comment:
Perhaps  split up the conditional statements to avoid long methods?

Reply:
Thanks for the suggestions! I've made the changes to imports but they're in a branch I haven't merged yet; I'll make sure to check the line wrapping as well before I next push.

Can I clarify what you mean by splitting up the conditional statements?